guides/common/modules/proc_cloning_satellite_server.adoc

```diff
@@ -143,10 +143,21 @@ If you copy to a different folder, update the `backup_dir` variable in the `/etc
 You can either mount the shared storage or copy the backup files to the `/backup/` folder on the target server.
 . Power off the source server.
 . Register your instance to the Red Hat Customer Portal and enable only the required repositories:
+** If the target server runs {EL}{nbsp}9, enter the following commands:
```

Comment:
This only matters if we use it outside of the Satellite context, but you have the explicit RHEL attributes so I'd be explicit:
```suggestion
** If the target server runs {RHEL}{nbsp}9, enter the following commands:
```

Reply:
Thanks! Applied here and on line 156 for EL/RHEL 8 too.